cala-server/src/graphql/tx_template.rs

```diff
@@ -0,0 +1,101 @@
+use async_graphql::*;
+
+use super::primitives::*;
+
+#[derive(SimpleObject)]
+pub(super) struct TxTemplate {
+    pub id: ID,
+    pub tx_template_id: UUID,
+    pub code: String,
+    pub params: Option<Vec<ParamDefinition>>,
```

Comment:
Why option? It’s a Vec so it can just be empty

Reply:
yep I was going to ask this question as to why we have this type in sqlx-ledger... I think we should change this to Vec<ParamDefinition>